docs/source/conf.py

```diff
@@ -84,6 +85,7 @@
         },
     ],
     "logo": {
+        # The release version can be included here if desired later by using f"{project} {release}"
         "text": f"{project}",
```

Comment:
```suggestion
        "text": f"{project} v{release}",
```
I thought it could be nice to include the version in the cookiecutter docs too (as we do in the template and in all our tools). However here the name becomes apparently too long, and annoyingly it overlaps with the section names when the window is narrow. The [issue](https://github.com/pydata/pydata-sphinx-theme/issues/1143) was reported in pydata-sphinx-theme a while ago, and is in theory fixed, but none of the hacks worked for me. I tried for a while but couldn't make it work 😢 Let me know if you find a nice fix, otherwise we can let it go!

Reply:
Ok sounds good, if the second options works better for a majority we may go for that. Thanks for the research!

Out of curiosity, what do you use to simulate different window sizes? Seems useful!

Reply:
I use the Browser  Developer Tools: Right-click and select "Inspect" any website.
Enable Device Mode and Change Dimensions to any device i want